package-whitelist.json

```diff
@@ -24,6 +24,8 @@
   "common/vendor/autoload_52.php",
   "common/vendor/composer/*",
   "common/vendor/lucatume/di52/src/**",
+  "common/vendor/select2/select2*.js",
+  "common/vendor/select2/select2.css",
```

Comment:
Oops! We need to also include the images.

Moving the `vendor/select2/*` lines from below up to here and prepending `common/` should catch the items that are missing here.